### PR TITLE
Add test step to basic tutorial

### PR DESCRIPTION
**Pull request by TyDunn**
**Proposed changes**:
- Add an eighth step about how you should run tests after training a new model

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by TyDunn** [comment]
@akelad @erohmensing I requested a review from both of you because it is small but significant change to the rasa tutorial, and I want to get it into the `rasa 1.9` release tomorrow

File: docs/user-guide/rasa-tutorial.rst [hunk 1]
```diff
@@ -223,8 +223,23 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the model
```
Comment (erohmensing) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
7. Test the Model
```

File: docs/user-guide/rasa-tutorial.rst [hunk 2]
```diff
@@ -223,8 +223,23 @@ different model parts if something has changed in their data or configuration.
...
+In Rasa, you use end-to-end tests defined in your ``tests/`` directory to run through
+test conversations that ensure both NLU and Core make correct predictions.
+
+.. runnable::
+
+   rasa test
+
+   echo "Finished running tests."
+
+See :ref:`evaluating-models` to find out how you can define more tests and learn about more ways you
```
Comment (erohmensing) [inline docs/user-guide/rasa-tutorial.rst]: what do you mean by "how you can define more tests? I'd probably just say `to learn more about how to evaluate your model as you improve it`

Comment (TyDunn) [reply docs/user-guide/rasa-tutorial.rst]: v good point, stole your copy

File: docs/user-guide/rasa-tutorial.rst [hunk 3]
```diff
@@ -223,8 +223,22 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the Model
+^^^^^^^^^^^^^^^^^
 
-7. Talk to Your Assistant
+After you train a model, you always want to check that your assistant still behaves as you expect.
+In Rasa, you use end-to-end tests defined in your ``tests/`` directory to run through
```
Comment (akelad) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
In Rasa Open Source, you use end-to-end tests defined in your ``tests/`` directory to run through
```

File: docs/user-guide/rasa-tutorial.rst [hunk 4]
```diff
@@ -223,8 +223,22 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the Model
```
Comment (TyDunn) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
7. Test the Assistant
```

File: docs/user-guide/rasa-tutorial.rst [hunk 5]
```diff
@@ -223,8 +223,22 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the Assistant
+^^^^^^^^^^^^^^^^^
```
Comment (TyDunn) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^
```

**Comment by TyDunn** [comment]
@erohmensing were you able to do this?

File: docs/user-guide/rasa-tutorial.rst [hunk 6]
```diff
@@ -223,8 +223,22 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the Assistant
```
Comment (TyDunn) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
7. Test Your Assistant
```

File: docs/user-guide/rasa-tutorial.rst [hunk 7]
```diff
@@ -223,8 +223,22 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test Your Assistant
+^^^^^^^^^^^^^^^^^^^^^
```
Comment (TyDunn) [inline docs/user-guide/rasa-tutorial.rst]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by erohmensing** [comment]
No I wasn't able to do this yet, as I wasn't able to get it working locally due to branch referencing not working. @tmbo why did we merge this?

**Pull request merged**